docs/vsphere-golden-image.md

```diff
@@ -0,0 +1,172 @@
+# Creating the vSphere Windows VM Golden Image
+
+This guide describes the thought process of creating a Windows virtual machine base image (the Golden Image) in vSphere.
+
+
+## 1. Select a compatible Windows Server version
+
+Currently the Windows Machine Config Operator (WMCO) stable version [2.0.0](REF?) only supports Windows Server version 1909 (build 18363.\*) Semi-Annual Channel, that includes the patch [KB4565351](https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351), required to get the operating system feature that allows setting the [hybrid OVN Kubernetes networking with a custom VXLAN port](setup-hybrid-OVNKubernetes-cluster.md#vSphere)
+
+*Please note that the above patch is not available for Windows Server version 2019.*
+
+> Comming soon: Support for Windows Server versions 2004 (build 19041.\*) and 20H2 (build 19042.\*) Semi-Annual Channels.
```

Comment:
This is the point of the PR. Please remove and moreover no need to mention 20H2.